test/base.js

```diff
@@ -242,47 +240,47 @@ describe('Basic => ', function() {
 
         });
 
-        describe('a manual increment field => ', function() {
+        describe('a manual increment field => ', function () {
```

Comment:
Can you add a test case for manual, non-exclusive counters?

Reply:
ok